src/graphics.rs

```diff
@@ -1328,8 +1331,10 @@ impl<'a> SmartComponent<ScContext<'a>> for SpriteId {}
 #[derive(Debug)]
 struct SpriteBatchInner {
     instances: Vec<InstanceProperties>,
+    /// Capacity it used to store the length of the buffers inside of mq::Bindings
     capacity: usize,
     bindings: mq::Bindings,
+    /// A bounding box that encaspulates all of the sprites within the batch
```

Comment:
Specifically a cached bounding box which covers all sprites in the batch, wouldn't need this field if we weren't caching it